frontend/packages/dev-console/src/components/pipelines/__tests__/PipelinesPage.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineModel } from '../../../models';
+import { PipelinesPage } from '../PipelinesPage';
+import ProjectListPage from '../../projects/ProjectListPage';
+import PipelinesResourceList from '../PipelinesResourceList';
+
+const pipelinePageProps = {
+  history: null,
+  location: null,
+  match: {
+    isExact: true,
+    path: `/k8s/ns/:ns/${referenceForModel(PipelineModel)}`,
+    url: 'k8s/ns/my-project/tekton.dev~v1alpha1~Pipeline',
+    params: {
+      ns: 'my-project',
+    },
+  },
+};
+
+const pipelinePagePropsWNS = {
```

Comment:
Add type here

Reply:
I think you can move `pipelinePageProps` and `pipelinePagePropsWNS` inside their respective tests

Reply:
done